src/states/SelectMoveState.py

```diff
@@ -67,6 +68,20 @@ def Enter(self, params):
         print(f'Owner: {self.effectOwner}')
         print(f'Effect: {self.effect.type} ({self.effect.minRange} - {self.effect.maxRange})')
         print(f'SelectMoveTile: {self.selectMoveTile}')
+
+        if self.effectOwner == PlayerType.ENEMY:
+            randomLeft = []
+            randomRight = []
+            for index in range(len(self.avilableMoveTile)):
+                if not self.field[self.avilableMoveTile[index]].is_occupied():
+                    if self.avilableMoveTile[index] <= self.enemy.fieldTile_index:
+                        randomLeft.append(index)
+                    if self.avilableMoveTile[index] >= self.enemy.fieldTile_index:
+                        randomRight.append(index)
+            if self.enemy.fieldTile_index > self.player.fieldTile_index:
+                self.selectMoveTile = random.choice(randomLeft)
+            else:
+                self.selectMoveTile = random.choice(randomRight)
```

Comment:
What a pure random move hahaha. Nice one.

Reply:
this could be use for confusion debuff in the future